<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="okPushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="mainStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="numReplacePage">
     <widget class="QSpinBox" name="zeroNumSpinBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>向前补零：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="beginNumSpinBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>起始数字：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>数字间隔：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="numIntervalSpinBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="partReplacePage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>查找：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>替换为：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="findLineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="replaceLineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="IgnoreExCheckBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>忽略扩展名</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="appendPage">
     <widget class="QCheckBox" name="suffixCheckBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>向后追加</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="prefixCheckBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>向前追加</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>追加：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="appendLineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QComboBox" name="numTypeComboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1,2,3,4,5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>-1,-2,-3,-4,-5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>_1,_2,_3,_4,_5</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>86</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>序号样式：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="filesListWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>131</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>439</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>替换</string>
    </property>
    <addaction name="partReplaceAction"/>
    <addaction name="numReplaceAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>追加</string>
    </property>
    <addaction name="diyAppendAction"/>
    <addaction name="numAppendAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="seleteFileAction"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="diyAppendAction">
   <property name="text">
    <string>自定义</string>
   </property>
  </action>
  <action name="numAppendAction">
   <property name="text">
    <string>序号</string>
   </property>
  </action>
  <action name="partReplaceAction">
   <property name="text">
    <string>部分</string>
   </property>
  </action>
  <action name="numReplaceAction">
   <property name="text">
    <string>序号</string>
   </property>
  </action>
  <action name="seleteFileAction">
   <property name="text">
    <string>选择文件</string>
   </property>
   <property name="toolTip">
    <string>seleteFile</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
